### PR TITLE
Passing Sort and Order options to Export Buttons

### PR DESCRIPTION
**Pull request by yungtechboy1**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A

### CODE REVIEW

**Review by trasher — APPROVED** [review]

File: templates/components/search/controls.html.twig [hunk 1]
```diff
@@ -101,7 +101,7 @@
                <i id="export_dropdown_icon" class="ti fa-lg ti-file-download"></i>
             </span>
          </button>
-      {% set exporthref = path('/front/report.dynamic.php') ~ '?item_type=' ~ itemtype ~ '&start=' ~ start ~ '&' ~ posthref %}
+      {% set exporthref = path('/front/report.dynamic.php') ~ '?item_type=' ~ itemtype ~ '&sort[]=' ~ sort[0] ~ '&order[]=' ~ order[0]  ~ '&start=' ~ start ~ '&' ~ posthref %}
```
Comment (trasher) [inline templates/components/search/controls.html.twig:104]:
```suggestion
      {% set exporthref = path('/front/report.dynamic.php') ~ '?item_type=' ~ itemtype ~ '&sort[]=' ~ sort[0] ~ '&order[]=' ~ order[0] ~ '&start=' ~ start ~ '&' ~ posthref %}
```

File: templates/components/search/controls.html.twig [hunk 2]
```diff
@@ -101,7 +101,7 @@
                <i id="export_dropdown_icon" class="ti fa-lg ti-file-download"></i>
             </span>
          </button>
-      {% set exporthref = path('/front/report.dynamic.php') ~ '?item_type=' ~ itemtype ~ '&start=' ~ start ~ '&' ~ posthref %}
+      {% set exporthref = path('/front/report.dynamic.php') ~ '?item_type=' ~ itemtype ~ '&sort[]=' ~ sort[0] ~ '&order[]=' ~ order[0] ~ '&start=' ~ start ~ '&' ~ posthref %}
```
Comment (orthagh) [inline templates/components/search/controls.html.twig:104]:
```suggestion
      {% set exporthref = path('/front/report.dynamic.php') ~ {
        'item_type': itemtype,
        'sort': sort,
        'order': order,
        'start': start,
      }|url_encode ~ posthref %}
```

This to have also multisort feature included in the href

Comment (trasher) [reply templates/components/search/controls.html.twig:104]: ping @yungtechboy1 ?

**Comment by cedric-anne** [comment]
@yungtechboy1 

Could you explain the functionnal impact and where these parameters are used ?

File: templates/components/search/controls.html.twig (hunk 2 above)
Comment (yungtechboy1) [reply templates/components/search/controls.html.twig:104]: This works also, just need 1 small change...

**Comment by yungtechboy1** [comment]
The purpose of this PR is to fix a Bug where your search filter and order values are not passed to  report.dynamic.php.
@cedric-anne

**Review by orthagh — APPROVED** [review]

**Review by trasher — APPROVED** [review]

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**